File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        SortedSet<Integer> ss=new TreeSet<>();
        for(int i=0;i<nums.length;i++){
            ss.add(nums[i]);
        }
       int i = 0;

for (int val : ss) {
    nums[i++] = val;
}
        int k=ss.size();
        return k;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n=nums.size(); int k=1; int p=1; int mn=nums[0];
        for(int i=1;i<n;i++){
            if(mn<nums[i]){
                mn=nums[i];
                k++;
                swap(nums[p],nums[i]);
                p++;
            }
        }
        return k;
    }
};